### PR TITLE
[RFC][UN-9887] Add block opt to select and update step card style

### PR DESCRIPTION
**Pull request by henrmota**
https://uniplaces.atlassian.net/browse/UN-9887

dependency on: https://github.com/uniplaces/uniplaces-uniq/pull/137

### CODE REVIEW

File: addon/styles/partials/_uni-step-card.scss [hunk 1]
```diff
@@ -5,6 +5,62 @@
         width: 100%;
     }
 
+    text-align: center;
```
Comment (AndreJoaquim) [inline addon/styles/partials/_uni-step-card.scss]: This changes should be on uniQ

Comment (AndreJoaquim) [reply addon/styles/partials/_uni-step-card.scss]: [Here](https://github.com/uniplaces/uniplaces-uniq/blob/master/src/styles/_components/_step-card.scss)

**Review by AndreJoaquim — CHANGES_REQUESTED** [review]

File: addon/styles/partials/_uni-step-card.scss [hunk 2]
```diff
@@ -5,6 +5,34 @@
...
+            }
+
+            svg g {
+                fill: none !important;
+            }
+        }
+    }
+
+    &__icon {
+
```
Comment (AndreJoaquim) [inline addon/styles/partials/_uni-step-card.scss]: Remove empty line

File: addon/styles/partials/_uni-step-card.scss [hunk 3]
```diff
@@ -5,6 +5,34 @@
...
+      svg {
+          position: static !important;
+          width: 146px !important;
+          height: auto !important;
+      }
+    }
+
+    .uni-button{
```
Comment (AndreJoaquim) [inline addon/styles/partials/_uni-step-card.scss]: Missing space

File: addon/styles/partials/_uni-step-card.scss [hunk 4]
```diff
@@ -5,6 +5,34 @@
         width: 100%;
     }
 
+    &--passed {
```
Comment (AndreJoaquim) [inline addon/styles/partials/_uni-step-card.scss]: Should this be part of our CSS framework (uniq), @mmelo ?

**Review by AndreJoaquim — CHANGES_REQUESTED** [review]

File: addon/styles/partials/_uni-step-card.scss (hunk 4 above)
Comment (mmelo) [reply addon/styles/partials/_uni-step-card.scss]: not wanting to nitpick, I must agree with @AndreJoaquim, this should be on uniQ (css) as it, if used elsewhere should have the passed state colors according to the rest of the changes made here. 🎖

**Review by mmelo — APPROVED** [review]

**Review by AndreJoaquim — APPROVED** [review]

**Pull request merged**